Don't lint node_modules

josephfrazier opened the pull request.

stefanbuck commented:
Eslint takes the .gitignore into account so I wonder why this is an issue for you.

josephfrazier commented:
Yeah... I'm not really sure why it just started happening for me. However, it seems like while [eslint is supposed to ignore node_modules by default](https://eslint.org/docs/3.0.0/user-guide/configuring#ignoring-files-and-directories), it [doesn't obey .gitignore by default](https://eslint.org/docs/3.0.0/user-guide/configuring#using-an-alternate-file). I'll keep thinking about this.

josephfrazier commented:
Oh, I figured it out. I had previously created a branch to prepare for merging this repo into the https://github.com/OctoLinker/OctoLinker monorepo, and on that branch I had moved everything into `./packages/live-resolver`. When I switched back to the `master` branch, the `packages` directory stayed on disk, so eslint tried to process `./packages/live-resolver/node_modules/`
